frontend/src/lib/i18n/locales/en.json

```
@@ -242,12 +242,19 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
       "log_header": "Message"
     },
     "get_project": {
-      "instructions_header": "To open the project with{type, select, FL_EX { } other { (e.g.) }}FLEx{mode, select, manual { manually} other { }}",
-      "instructions": "1. In the \"Send/Receive\" menu click \"Get Project from colleague…\"\n\
+      "instructions_header": "To open the project with {type, select, FL_EX { FLEx } WE_SAY { WeSay } other { (e.g.) FLEx } } {mode, select, manual { manually} other { }}",
+      "instructions_flex": "1. In the \"Send/Receive\" menu click \"Get Project from colleague…\"\n\
 1. In the \"Receive project\" dialog box, click \"Internet\"\n\
 1. In the \"Get Project From Internet\" dialog box, enter your \"Login\" and \"Password\" for Language Depot and click \"Log in\"\n\
 1. In the \"Project ID\" field that appears, select \"{code}\"\n\
 1. Enter a \"Name for the folder on your computer\" (e.g. \"{name}\")",
+      "instructions_wesay": "1. Launch 'WeSay Configuration Tool'\n\
+1. Click 'Get from Internet'\n\
+1. In the 'Project ID' field type: \"{code}\"\n\
+1. In the 'Login' field type: \"{login}\"\ Note: there is a bug in WeSay that means you can't type the @ symbol for an email, this is a workaround \n\
```

```suggestion
1. In the 'Login' field type: \"{login}\"\ Note: A bug in WeSay means you can't type the @ symbol for an email.  The workaround is to type %40 in place of @. \n\
```

I'm not sure if I need to escape the % sign here.